src/Tools/ExternalAccess/Format/ImportsOrganizer.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.OrganizeImports;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Format
+{
+    internal static class ImportsOrganizer
+    {
+        public static Task<Document> OrganizeImportsAsync(Document document, CancellationToken cancellationToken)
+            => OrganizeImportsService.OrganizeImportsAsync(document, cancellationToken);
+    }
+}
```

Comment:
is this class necessary?  it doesn't seem to be doing anything other than forwarding...

Reply:
That is right. Since only ExternalAccess libraries can have IVTs to Roslyn this type exists to forward calls from dotnet-format to the OrganizeImportsService.